common/koa-middleware/withPopupDialog.js

```diff
@@ -0,0 +1,29 @@
+const Prismic = require('prismic-javascript');
+
+let popupDialog = {
+  openButtonText: null,
+  dialogHeading: null,
```

Comment:
any reason we're not just going with
* `heading` or `title`
* `copy`
* `linkText`
* `link`

?